tests/PHPUnit/Unit/Validator/AlphaNumSpacesTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Unit\Translation\Loader;
+
+use PHPUnit\Framework\TestCase;
+use Piwik\Validators\AlphaNumSpaces;
+
+/**
+ * @group Validator
+ * @group AlphaNumSpaces
+ * @group AlphaNumSpacesTest
+ */
+class AlphaNumSpacesTest extends TestCase
+{
+    public function test_validate_success_english_strings()
+    {
+        $this->validate('onlyalpha');
+        $this->validate(123);
+        $this->validate('abc 123');
+        $this->validate('ABC 123 and 0');
+
+        $this->assertTrue(true);
```

Comment:
might be better to use `$this->expectNotToPerformAssertions();` instead of doing an assert that can't fail.
Also you could also use one test for all strings with a dataprovider, like for the failing ones.